### ISSUE TITLE
German localization

### ISSUE THREAD

**Issue by lothargrieb** [moment/moment#2649]
Hi,
the short strings for german (de) should look like this:

```
monthsShort : 'Jan_Feb_Mrz_Apr_Mai_Jun_Jul_Aug_Sep_Okt_Nov_Dez'
weekdaysShort : 'So_Mo_Di_Mi_Do_Fr_Sa'
```

The current version with dots looking very strange and are not required.

Almost same for Austria (de_at):

```
monthsShort : 'Jän_Feb_Mär_Apr_Mai_Jun_Jul_Aug_Sept_Okt_Nov_Dez'
weekdaysShort : 'So_Mo_Di_Mi_Do_Fr_Sa'
```

Many thanks in advance.

**Comment by alburthoffman** [comment]
You are right. One PR would be good to fix this.

**Comment by HoffmannP** [comment]
The German norm DIN 5008 recommends "Mär" instead of  "Mrz". But also recommends "Sept" instead of "Sep". Confusing!
Also, it allowes for months with only four characters to not be abreviated (eg. Juli instead of Jul.)

http://www.din-5008-richtlinien.de/datum.php

I'm not quite sure which option should be implemented :-(

**Comment by Oire** [comment]
@lothargrieb, would you submit a PR to get this fixed, please?

**Comment by pomm0** [comment]
+1

**Comment by marwahaha** [comment]
Based on the discussion in #3803 and the fix in #3829 , I'm going to close this. If you think this merits more discussion, please re-open @lothargrieb
